generate_sample_file.py

```diff
@@ -101,9 +101,23 @@ def get_random_post_town(self):
     def get_random_post_code(self):
         first_random_number = random.randint(1, 9)
         second_random_number = random.randint(1, 9)
-        return f'{self.get_random_letter()}{self.get_random_letter()}{first_random_number} {second_random_number}' + \
+        return f'{self.get_random_letter()}{self.get_random_letter()}{first_random_number} {second_random_number}' \
                f'{self.get_random_letter()}{self.get_random_letter()}'
 
+    def get_random_field_coordinator_id(self):
```

Comment:
why not use this function in `get_random_field_officer_id` to save code duplication? Anyway... just minor stuff

Reply:
In my mind, I wanted to distinguish the difference between the different formats of the two fields